modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js

```diff
@@ -285,93 +311,128 @@ const InviteTeamMembersPage = ({
-				<div className="mx-3 pt-3">
-					<h5 className="text-neutral-7">
-						{`${
-							projectHasSLAGoldPlatinum
-								? ROLE_TYPES.requestor.name
-								: ROLE_TYPES.admin.name
-						}	roles available: ${availableAdminsRoles} of ${maxRequestors}`}
-					</h5>
-
-					<p className="mb-0 text-neutral-7 text-paragraph-sm">
-						{`Only ${maxRequestors} member${
-							maxRequestors > 1 ? 's' : ''
-						} per project (including yourself) have
-						 role permissions (Admins & Requestors) to open Support
-						 tickets. `}
-
-						<a
-							className="font-weight-bold text-neutral-9"
-							href={supportLink}
-							rel="noreferrer"
-							target="_blank"
-						>
-							Learn more about Customer Portal roles
-						</a>
-					</p>
-				</div>
-			</div>
+			<FieldArray
+				name="invites"
+				render={({pop, push}) => (
+					<>
+						<div className="invites-form overflow-auto px-3">
+							<div className="px-3">
+								<label>Project Name</label>
+
+								<p className="invites-project-name text-neutral-6 text-paragraph-lg">
+									<strong>{project.name}</strong>
+								</p>
+							</div>
+
+							<ClayForm.Group className="m-0">
+								{values?.invites?.map((invite, index) => (
+									<TeamMemberInputs
+										disableError={hasInitialError}
+										id={index}
+										invite={invite}
+										key={index}
+										options={accountRolesOptions}
+										placeholderEmail={`username@${
+											project?.code?.toLowerCase() ||
+											'example'
+										}.com`}
+										selectOnChange={(roleId) =>
+											setFieldValue(
+												`invites[${index}].role`,
+												accountRoles?.find(
+													({id}) => id === +roleId
+												)
+											)
+										}
+									/>
+								))}
+							</ClayForm.Group>
+
+							{showEmptyEmailError && (
+								<Badge>
+									<span className="pl-1">
+										Please enter your email address.
+									</span>
+								</Badge>
+							)}
+
+							<div className="ml-3 my-4">
+								{values?.invites?.length > 1 && (
+									<Button
+										className="mr-3 py-2 text-brandy-secondary"
+										displayType="secondary"
+										onClick={() => {
+											const removedItem = pop();
+
+											if (
+												removedItem.role.name ===
+													'Administrator' ||
+												removedItem.role.name ===
+													'Requestor'
+											) {
+												setAvailableAdminsRoles(
+													(previousAdmins) =>
+														previousAdmins + 1
+												);
+											}
+										}}
+										prependIcon="hr"
+										small
+									>
+										Remove this Member
+									</Button>
+								)}
+
+								{values?.invites?.length <
+									MAXIMUM_INVITES_COUNT && (
+									<Button
+										className="btn-outline-primary cp-btn-add-members py-2 rounded-xs"
+										onClick={() => {
+											setBaseButtonDisabled(false);
+
+											if (handleAddTeamMember()) {
+												push(
```

Comment:
Attach the value in some variable before use within if.
And should be good choose a better name for this function. What the purpose? I see that it's return a validation and also set in some state.

Function need to be scoped. A better approach it's rename this function to isEmptyFillEmails, or change it to a simple variable, and the state's update come inside this condition